delivery_package_number/README.rst

```diff
@@ -0,0 +1,91 @@
+============================
+Stock Picking Package Number
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery-lightgray.png?logo=github
+    :target: https://github.com/OCA/delivery/tree/12.0/delivery_package_number
+    :alt: OCA/delivery
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/delivery-12-0/delivery-12-0-delivery_package_number
+    :alt: Translate me on Weblate
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module adds a number of packages field that can be either be set manually
+by the user or computed when the delivery packages flow is used.
+
+This field can be used by delivery carrier extensions that need such info to
+rate the shipment, print the labels, etc.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+If no delivery packages are used:
+
+#. Go to an open picking and click in the *Additional Info* tab.
+#. In the *Delivey Information* section you'll find a **Number of packages**
+   field that you can edit.
+
+If delivey packages are used:
+
+#. The field will be computed depending on the delivery packages used in the
```

Comment:
I think the field should always remain writable. The only change using delivery packages is that the field gets auto-filled when you change these packages, but you should be able to change the number of packages anyway. Example of use case: imagine you set packages for an specific product, but the rest are put randomly in other packages and you want to set this manually.